cli/stream_command.go

```diff
@@ -1775,6 +1848,15 @@ func (c *streamCmd) showStreamInfo(info *api.StreamInfo) {
 
 	showSource := func(s *api.StreamSourceInfo) {
 		cols.AddRow("Stream Name", s.Name)
+		if s.SubjectTransformDest != "" {
+			var filter = ">"
```

Comment:
Got it and cleaned up at this and other places. Apologies my code may contain leftover reflexes from using a functional language with `val` and `var` (rather than `const`) and bleeding into my Go code where I use `var` to signify 'mutable' which is obviously lost in the translation :).